### PR TITLE
Add jupyterlab-daw

### PR DESCRIPTION
**Pull request by benbovy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by benbovy** [comment]
@conda-forge/help-python, this is ready for review.

File: recipes/jupyterlab-daw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  commands:
+    - pip check
+    - jupyter labextension list
+    - jupyter labextension list 1>labextensions 2>&1
+    - grep -iE "jupyterlab-daw.*v{{ version.replace(".", "\\.") }}.*enabled.*OK.*jupyterlab_daw" labextensions
+
+about:
+  home: https://github.com/benbovy/jupyterlab-daw
+  license: BSD-3-Clause
+  license_family: BSD
```
Comment (dopplershift) [inline recipes/jupyterlab-daw/meta.yaml:45]: Based on what's in the 3rd party directory, it's really:
```suggestion
  license: BSD-3-Clause AND MIT AND 0BSD
```

Comment (benbovy) [reply recipes/jupyterlab-daw/meta.yaml:45]: Is it common for the `license` entry to have multiple licenses? Other jupyter extension recipes in conda-forge don't seem to include the license names of the 3rd-party JS dependencies (https://github.com/search?q=org%3Aconda-forge+third-party-licenses.json&type=code&p=1), but I'm happy to apply your suggested change if that's better!

Comment (dopplershift) [reply recipes/jupyterlab-daw/meta.yaml:45]: Well, I wasn't aware of that file being buried deep in the package files. In an ideal world, they probably should all do this. In this case, at the bare minimum the `license` field should match what's being linked in `license_file`.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**